tree/ntuple/v7/inc/ROOT/RMiniFile.hxx

```diff
@@ -112,6 +112,10 @@ public:
    /// The key length of a blob. It is always a big key (version > 1000) with class name RBlob.
    static constexpr std::size_t kBlobKeyLen = 42;
 
+   /// Prepares buffer for a new record as an RBlob key at offset.
+   /// (Note that the array type is purely documentation, the argument is actually just a pointer.)
+   static void PrepareBlobKey(std::int64_t offset, size_t nbytes, size_t len, unsigned char buffer[kBlobKeyLen]);
```

Comment:
This method should be declared further down, after the private methods.

Reply:
Hm, not sure: `PrepareBlobKey` is some sort of `static` utility method, while the other two `public` functions `Recreate` and `Append` are factory methods that actually create a `RNTupleFileWriter`. I think I would prefer to keep the declaration close to that of `kBlobKeyLen`.